src/payment/strategies/klarna/klarna-payment-strategy.ts

```diff
@@ -121,6 +133,35 @@ export default class KlarnaPaymentStrategy implements PaymentStrategy {
             }));
     }
 
+    private _getUpdateSessionData(billingAddress: BillingAddress, shippingAddress?: Address): KlarnaUpdateSessionParams {
+        if (this._supportedEUCountries.indexOf(billingAddress.countryCode) === -1) {
```

Comment:
Please, use `lodash#includes` it is easier to understand what's going on.

```suggestion
        if (includes(this._supportedEUCountries, billingAddress.countryCode)) {
```

You will need to import it.